packages/perseus/src/perseus-types.ts

```diff
@@ -653,21 +653,25 @@ export type PerseusInteractiveGraphWidgetOptions = {
 };
 
 const lockedFigureColorNames = [
+    "blue",
     "green",
     "grayH",
     "purple",
     "pink",
+    "orange",
     "red",
 ] as const;
 
 export type LockedFigureColor = (typeof lockedFigureColorNames)[number];
 
 export const lockedFigureColors: Record<LockedFigureColor, string> = {
+    blue: "#3D7586",
```

Comment:
Question: Are these the same as the WB color tokens? Just curious :)

Reply:
Nope! They're MathJax colors.